plugin_tooling/src-tests/com/github/rustdt/tooling/RustBuildOutputParserTest.java

```diff
@@ -49,135 +29,4 @@ protected static ToolSourceMessage msg(Path path, int line, int column, int endL
 		msg.toString();
 		return msg;
 	}
```

Comment:
I moved the code that is specific to the "classical" parser (rustc 1.11 or below) to the file RustBuildOutputParserClassicTest.java.